docs/high-level-design.md

```diff
@@ -4,38 +4,38 @@ Common LIMS is a framework to create custom LIMS systems in a consistent and sim
 
 The approach behind Common LIMS is to assume that it will be used in wildly different organizations. To approach this, we split the system in three sections:
 
-	* Core
-		* Addresses features you would expect from any information system
-		* Related UI components
-	* Lab
-		* Provides APIs for features that are common amongst most or all labs
-		* Related UI components
-	* Plugins
-		* Workflows
-		* Custom code
+    * Core
+    	* Addresses features you would expect from any information system
+    	* Related UI components
+    * Lab
+    	* Provides APIs for features that are common amongst most or all labs
+    	* Related UI components
+    * Plugins
+    	* Custom code
```

Comment:
Misses the "Workflows", perhaps unintentionally. But the workflows are defined in the plugins.

Reply:
Yeah. I though that referring to it here with out first having defined what it means could make the text more difficult to understand.

Reply:
Right, I guess we'll have to revisit this document several times. But I wonder if we shouldn't then restructure it a bit. 

It also depends on the audience reading this. When I wrote it I was thinking of devs, which would know what workflows are. It might make sense to read the documentation together at some later point and brainstorm on ideas.

Reply:
I actually think that there are a wide variety of different understandings of what a workflow is, especially if you deal with people who have experience in bioinformatics. But I don't think that this is critical at this point, as you say we will need to get back to this later.